DemoApp/Sources/Views/CallView/CallingView/DetailedCallingView.swift

```diff
@@ -8,233 +8,214 @@ import StreamVideoSwiftUI
 import SwiftUI
 
 struct DetailedCallingView: View {
-    @ObservedObject var viewModel: CallViewModel
-
     @Injected(\.streamVideo) var streamVideo
+    @Injected(\.appearance) var appearance
 
-    private let imageSize: CGFloat = 32
-
-    @State private var callId = ""
-
-    @State private var callAction = CallAction.startCall
+    @ObservedObject var viewModel: CallViewModel
+    @ObservedObject private var appState = AppState.shared
 
-    @State private var callFlow: CallFlow = .joinImmediately
+    private let imageSize: CGFloat = 32
 
-    var participants: [User] {
+    private var participants: [User] {
```

Comment:
Not directly related to this PR, but I saw that this property could be shorter. But that is nothing what we should really fix in this PR.
```swift
private var participants: [User] {
  AppState.shared.users.filter { $0.id != streamVideo.user.id }
}
```